.circleci/config.yml

```diff
@@ -92,7 +92,7 @@ jobs:
             ls -l ~/artifacts/*
       - run:
           name: "S3 Release"
-          command: aws s3 cp ~/artifacts s3://honeycomb-builds/honeycombio/honeycomb-lambda-extension/${CIRCLE_TAG}/ --recursive
+          command: aws s3 cp ~/artifacts/linux s3://honeycomb-builds/honeycombio/honeycomb-lambda-extension/${CIRCLE_TAG}/ --recursive
```

Comment:
Can you elaborate a bit more, please? Like, why doesn't this change need to be made in multiple places, and what is the actual effect of eliminating these 6 characters? Are they just meaningless? Why were they there in the first place?

Reply:
Yes! I should have elaborated here. If what I write below makes sense, I'll copy it to the PR description for merge-commit-message posterity.

Quick Answers: 

> why doesn't this change need to be made in multiple places

The change was previously made in multiple places (in #103 and in https://github.com/honeycombio/honeycomb-lambda-extension/pull/108/commits/53da4b25c493a860aa53efc25d1e4be9365e92af which was a follow-up change/fix like this one); this spot is a place that was missed.

> what is the actual effect of eliminating these 6 characters?

The built artifacts will appear directly in S3 under the `honeycomb-lambda-extension/<VERSION>` subdirectory of the honeycomb-builds bucket (which matches the contents for past releases) instead of in a `honeycomb-lambda-extension/<VERSION>/linux` subdirectory. This gets things back into alignment with [the download URL given in docs for retrieving the layer contents for use in custom Docker images](https://docs.honeycomb.io/getting-data-in/integrations/aws/aws-lambda/#installing-in-a-container-image).

> Are they just meaningless? Why were they there in the first place?

A longer story … 

A fair amount of build and release fiddly bits changed in #103. One major change was moving the compile/build and creation of ZIP files for layer publishing to the Makefile. Because [the compile/build can happen for any GOOS](https://github.com/honeycombio/honeycomb-lambda-extension/blob/v11.0.0/Makefile#L1-L5), I wanted to put guard rails up to prevent us from accidentally pushing up, say, a macOS darwin build in a published release layer. It only makes sense to publish linux-flavored layers. With all the building and zipping happening in the Makefile, I leveraged target file paths to sorta-kinda namespace compiled executables by their GOOS value and to declare the ZIP file targets as linux-only (in https://github.com/honeycombio/honeycomb-lambda-extension/pull/103/commits/46d7dbc9d0156e9c0f642ac26e26946a9f435e11).

So, GOOS appears in paths in the CI build environment, but any GOOS other than linux is useless/breaking in the context of _release_ artifacts. Therefore, the `publsh_*` jobs need to publish only linux things _and_ ought not to include "linux" in the paths for release artifacts because that's always been assumed.